src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "NETStandard.Library": "1.6.0",
+    "Newtonsoft.Json": "7.0.1",
+    "NuGet.Versioning": "3.5.0-beta2-1456"
```

Comment:
Not sure how you handle updating versions here, but the CLI preview2 shipped on NuGet 3.5.0-beta2-1484.